src/lms-harmonizer/tests_integration_sql/orchestrator.py

```diff
@@ -132,8 +132,13 @@ def _load_lms_extension_scripts(config: ServerConfig):
 
 
 def _load_ordered_scripts(config: ServerConfig, script_path: str):
+    files_in_path: List[str] = [
+        f
+        for f in os.listdir(script_path)
+        if os.path.isfile(os.path.join(script_path, f))
+    ]
```

Comment:
Good change, avoiding accidentally grabbing random sub-directories.

Reply:
Yeah, I got bit by that during testing. 😎